libs/common/src/enums/event-type.enum.ts

```diff
@@ -64,6 +64,7 @@ export enum EventType {
   Organization_EnabledKeyConnector = 1606,
   Organization_DisabledKeyConnector = 1607,
   Organization_SponsorshipsSynced = 1608,
+  Organization_CollectionManagemenet_Updated = 1609,
```

Comment:
typo in `Management`. Also, the format seems to be `<Domain>_<DescriptionInCamelCase>`, so I don't think you need the second underscore.

```suggestion
  Organization_CollectionManagementUpdated = 1609,
```

Reply:
I personally like the naming scheme shown, but not enough to defend it and stand out from the remaining 98% of enums. Updated!